system/cache/providers/LuceeProvider.cfc

```diff
@@ -111,10 +111,13 @@ component
      * Get a structure of all the keys in the cache with their appropriate metadata structures. This is used to build the reporting.[keyX->[metadataStructure]]
      */
     struct function getStoreMetadataReport(){
-		return getKeys()
-			.map( function( item ){
-				return getCachedObjectMetadata( item );
+		var result = {};
```

Comment:
Same as above, the `map()` function does the same.

Reply:
See above.